service/src/main/java/bio/terra/policy/service/policy/PolicyMutator.java

```diff
@@ -25,15 +25,19 @@ public class PolicyMutator {
    */
   public static PolicyInput combine(PolicyInput target, PolicyInput addPolicy) {
     validateMatchedPolicies(target, addPolicy);
-    return findPolicy(target).combine(target, addPolicy);
+    return findPolicy(target == null ? addPolicy : target).combine(target, addPolicy);
```

Comment:
Why not just pass in the addPolicy? The `findPolicy` is locating the right policy base for dispatching to the right `combine` so there is no benefit of one input vs the other.

Ditto on the removePolicy below.

Reply:
Yeah, that makes sense.